ASIM/dev/ASimTester/ASimTester.csv

```diff
@@ -1120,4 +1120,76 @@ ActorScope,string,Optional,ProcessEvent,,,
 TargetOriginalUserType,string,Optional,ProcessEvent,,,
 TargetScopeId,string,Optional,ProcessEvent,,,
 TargetScope,string,Optional,ProcessEvent,,,
-
+SrcIpAddr,string,Mandatory,Dhcp,IP Address,,
+IpAddr,string,Alias,Dhcp,IP Address,,SrcIpAddr
+RequestedIpAddr,string,Optional,Dhcp,IP Address,,
+SrcHostname,string,Mandatory,Dhcp,Hostname,,
+Hostname,string,Alias,Dhcp,,,DstHostname
+SrcDomain,string,Recommended,Dhcp,Domain,,
+SrcDomainType,string,Conditional,Dhcp,Enumerated,Windows|FQDN,SrcDomain
+SrcFQDN,string,Optional,Dhcp,FQDN,,
+SrcDvcId,string,Optional,Dhcp,,,
+SrcDvcScopeId,string,Optional,Dhcp,,,
+SrcDvcScope,string,Optional,Dhcp,,,
+SrcDvcIdType,string,Conditional,Dhcp,Enumerated,AzureResourceId|MDEid,SrcDvcId
+SrcDeviceType,string,Optional,Dhcp,Enumerated,Computer|Mobile Device|IOT Device|Other,
+SrcUserId,string,Optional,Dhcp,,,
+SrcUserIdType,string,Conditional,Dhcp,Enumerated,SID|UID|AADID|OktaId|AWSId,SrcUserId
+SrcUsername,string,Optional,Dhcp,,,
+Username,string,Alias,Dhcp,,,SrcUsername
+SrcUsernameType,string,Conditional,Dhcp,Enumerated,UPN|Windows|DN|Simple,SrcUsername
+SrcUserType,string,Optional,Dhcp,Enumerated,Regular|Machine|Admin|System|Application|Service Principal|Other,
+SrcOriginalUserType,string,Optional,Dhcp,,,
+SrcMacAddr,string,Mandatory,Dhcp,Mac Address,,
+DhcpLeaseDuration,int,Optional,Dhcp,,,
+DhcpSessionId,string,Optional,Dhcp,,,DhcpkSessionId
+SessionId,string,Alias,Dhcp,,,
+DhcpSessionDuration,int,Optional,Dhcp,,,
+Duration,string,Alias,Dhcp,,,DhcpSessionDuration
+DhcpSrcDHCId,string,Optional,Dhcp,,,
+DhcpCircuitId,string,Optional,Dhcp,,,
+DhcpSubscriberId,string,Optional,Dhcp,,,
+DhcpVendorClassId,string,Optional,Dhcp,,,
+DhcpVendorClass,string,Optional,Dhcp,,,
+DhcpUserClassId,string,Optional,Dhcp,,,
+DhcpUserClass,string,Optional,Dhcp,,,
+DvcAction,string,Optional,Dhcp,,,
+DvcDescription,string,Optional,Dhcp,,,
+DvcDomain,string,Recommended,Dhcp,Domain,,
+DvcDomainType,string,Conditional,Dhcp,Enumerated,Windows|FQDN|ResourceGroup,DvcDomain
+DvcFQDN,string,Optional,Dhcp,FQDN,,
+DvcHostname,string,Recommended,Dhcp,Hostname,,
+DvcId,string,Optional,Dhcp,,,
+DvcIdType,string,Conditional,Dhcp,Enumerated,AzureResourceId|MDEid|MD4IoTid|VMConnectionId|AwsVpcId|VectraId|AppGateId|Other,DvcId
```

Comment:
Not all of these DvcIdTypes are applicable to Dhcp. Please add only the once which are applicable. Otherwise just add AzureResourceId and Other.

Reply:
Okay, thank for taking the time to look atvthe PR.
Will update this tonight.